nix/sources.json

```diff
@@ -0,0 +1,26 @@
+{
+    "nixpkgs": {
+        "branch": "nixos-unstable",
+        "description": "Nix Packages collection",
+        "homepage": null,
+        "owner": "NixOS",
+        "repo": "nixpkgs",
+        "rev": "6c43a3495a11e261e5f41e5d7eda2d71dae1b2fe",
+        "sha256": "16f329z831bq7l3wn1dfvbkh95l2gcggdwn6rk3cisdmv2aa3189",
+        "type": "tarball",
+        "url": "https://github.com/NixOS/nixpkgs/archive/6c43a3495a11e261e5f41e5d7eda2d71dae1b2fe.tar.gz",
+        "url_template": "https://github.com/<owner>/<repo>/archive/<rev>.tar.gz"
+    },
+    "pyproject.nix": {
```

Comment:
Hm, doesn't look like this is actually used?

Reply:
Oops, good spotting. I was trying to use a nix package to handle the dynamic versioning and balked on it. Forgot to remove the dep. Fixing now.

Reply:
Fixed in 5e547ce